.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     parallelism: 1
     working_directory: ~/ezcater
     docker:
-      - image: circleci/ruby:2.4.1
+      - image: circleci/ruby:2.5.5
```

Comment:
any reason for 2.5 over other versions?

Reply:
I went with the latest 2.5.x release because I believe the majority of our apps are still on some version of 2.5.x.

I would have left this alone, but the postgresql client no longer installed on the old 2.4.1 base image, so bumping this was the easiest way to fix the build.